CHANGELOG.md

```diff
@@ -18,7 +18,23 @@ and this project does not adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.1] - 2014-11-15
 ### Added
-- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1 . 
+- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1. 
+
+## [0.2] - 2017-01-25
+### Added
+- New version of mex-wholebodymodel. A complete refactory of the toolbox has been done.
+
+### Renamed
+- WBM-wrappers functions have been renamed, and some of the outputs are listed in a different way.
```

Comment:
Yes, I would like to rewrite the entire CHANGELOG.md but as I said I was thinking to do it after merging, because I want to add some paths (e.g. path to new controllers, visualizer, ecc...) that won't exist anymore after merging. So even if I do it now, I will need to edit again the file after merging to adjust the paths.